deploy/bundle/catalogsource.yaml

```diff
@@ -0,0 +1,262 @@
+apiVersion: v1
+data:
+  clusterServiceVersions: |
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: ClusterServiceVersion
+      metadata:
+        name: windows-machine-config-operator.v0.0.1
+        namespace: placeholder
+        annotations:
+          alm-examples: >-
+            [{"apiVersion":"","kind":"WindowsMachineConfig","metadata":{"name":""},"spec":{}}]
+          categories: OpenShift Optional
+          certified: 'false'
+          createdAt: ''
+          description: >-
+            This operator allows for the addition of Windows nodes to an OpenShift
+            cluster.
+          containerImage: ''
+          support: ''
+          capabilities: Basic Install
+          repository: ''
+      spec:
+        displayName: Windows Machine Config
+        description: |
+          ## About the managed application
+          ## About this Operator
+          ## Prerequisites for enabling this Operator
+        maturity: alpha
+        version: 0.0.1
+        replaces: ''
+        skips: []
+        minKubeVersion: ''
+        keywords: []
+        maintainers:
+          - name: ''
+            email: ''
+        provider:
+          name: Red Hat
+        labels: {}
+        selector:
+          matchLabels: {}
+        links:
+          - name: Repository
+            url: 'https://github.com/openshift/windows-machine-config-operator'
+        icon:
+          - base64data: ''
+            mediatype: ''
+        customresourcedefinitions:
+          owned:
+            - name: windowsmachineconfigs.wmc.openshift.io
+              displayName: WindowsMachineConfig
+              kind: WindowsMachineConfig
+              version: v1alpha1
+              description: Windows Machine Config
+              resources:
+                - version: v1
+                  kind: Deployment
```

Comment:
Not sure where you're getting all these resources from?

Reply:
From deploying the operator and seeing what it created. I need to remove secret and configmap